src/main/resources/application-local.yml

```diff
@@ -0,0 +1,17 @@
+community-api:
+  base-url: http://localhost:8096
+
+spring:
+  security:
+    oauth2:
+      client:
+        registration:
+          nomis-oauth-client:
+            client-name: nomis-oauth-client
+            client-id: community-api-client
+            client-secret: clientsecret
```

Comment:
can you change this to a placeholder value with the default value of `clientsecret` to prevent any temptation to hard code secrets in config

Reply:
Good call, will do